evennia/server/inputfuncs.py

```diff
@@ -650,6 +651,26 @@ def msdp_send(session, *args, **kwargs):
     session.msg(send=((), out))
 
 
+def is_typing_get_aliases(session, *args, **kwargs):
+    session.msg(is_typing={'type': 'aliases', 'payload': CmdSay.aliases})
+
+
+def is_typing_state(session, *args, **kwargs):
+    # audience = ObjectDB.objects.filter(db_typeclass_path="typeclasses.characters.Character",
+    #                                    db_location=session.puppet.location).exclude(db_key=session.puppet.key)
+
+    audience = ObjectDB.objects.filter(db_typeclass_path="typeclasses.characters.Character",
```

Comment:
Only filtering on Characters is a very limited thing since devs may well change the typeclass they use for characters. Maybe you are rather looking for `DefaultCharacter.filter_family(...)` here?